drivers/scheduler/k8s/k8s.go

```diff
@@ -1461,6 +1465,47 @@ func (k *k8s) StartSchedOnNode(n node.Node) error {
 	return nil
 }
 
+// CreateCRDObjects and Validate their deployment
+func (k *k8s) CreateCRDObjects(ctx *scheduler.Context, timeout, retryInterval time.Duration) error {
+	var err error
+	k8sOps := k8s_ops.Instance()
+	for _, specObj := range ctx.App.SpecList {
+		if obj, ok := specObj.(*stork_api.ClusterPair); ok {
+			logrus.Info("Applying clusterpair spec")
+			err = k8sOps.CreateClusterPair(obj)
+			if err != nil {
+				logrus.Errorf("Error Creating Cluster Pair %v", err)
+				return err
+			}
+			if err := k8sOps.ValidateClusterPair(obj.Name, defaultTimeout, defaultRetryInterval); err != nil {
+				return &scheduler.ErrFailedToApplyCustomSpec{
+					Name:  obj.Name,
+					Cause: fmt.Sprintf("Failed to validate cluster Pair: %v. Err: %v", obj.Name, err),
```

Comment:
Add type to the error. No need to print clusterPair in the message. Same for migration below.